src/services/completions.ts

```diff
@@ -2543,6 +2543,15 @@ namespace ts.Completions {
             }
 
             if (contextToken.kind === SyntaxKind.GreaterThanToken && contextToken.parent) {
+                // <Component<string> /**/ />

```

Comment:
on its own, this fix makes sense to me. How does it relate to the next comment, which says there are two possibilities -- jsxtext inside a `<div>` and non-jsxtext inside a type-argument-having `<Component<T>`? It seems like the second case might be handled entirely by the new code, but I'm not sure. 

If it is, then the following check could be simplified.

Reply:
I thought the same thing, but removing it breaks the test added in the original "don't do completions in JsxText" PR; I can't remember if I figured out why or not at this point, though, so can go double check.

Reply:
Seems like it's the comment that needs to be updated; the second listed possibility is no longer accurate as `location` in this case isn't `GreaterThanToken`, it's the entire element. I'll change the comment to reflect that it's only handling the first case.